### PR TITLE
Create BLadd24988.xml

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: LondonBritishLibrary/add/BLadd24988.xml [hunk 1]
```diff
@@ -0,0 +1,211 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+   schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:id="BLadd24988" xml:lang="en">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>On the Councils, hāymānota ʾabaw, hāymānotu la-qǝddus ʿābiyy māri Yāʿqob ʾƎlbarādʿi</title>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/add/BLadd24988.xml:8]: Hāymānota

File: LondonBritishLibrary/add/BLadd24988.xml [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+                        </altIdentifier>
+                    </msIdentifier>
+
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                          <locus from="1r"></locus>
+                          <title type="complete" ref="LIT1601Histor" xml:lang="gez">
+                            <certainty locus="value" match="../@ref" cert="low"></certainty>
+                            SEction on the Councils of the Church
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/add/BLadd24988.xml:46]: Section

File: LondonBritishLibrary/add/BLadd24988.xml (hunk 1 above)
Comment (DariaElagina) [reply LondonBritishLibrary/add/BLadd24988.xml:8]: you it should be caputalized?

Comment (DenisNosnitsin1970) [reply LondonBritishLibrary/add/BLadd24988.xml:8]: I think both Haymanota and Haymanotu.

**Pull request merged**